lib/src/sync_package_resolver/index.dart

```diff
@@ -0,0 +1,3 @@
+export 'interface.dart'
```

Comment:
To match the way we stub out IO, this file (`sync_package_resolver/index.dart`) should be `sync_package_resolver.dart` instead.